python/tests/test_filesystem.py

```
@@ -177,7 +218,29 @@ def test_mkdir_flags():
   assert status.ok
   c.rm('/tmp/dir1/dir2')
   c.rm('/tmp/dir1')
-  
+
+def test_mkdir_p():
+  c = client.FileSystem(SERVER_URL)
+  status, response = c.mkdir_p('/tmp/dir1/dir2', flags=MkDirFlags.NONE)
+  assert status.ok
+  c.rm('/tmp/dir1/dir2')
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/dir1/dir2"

(S108)